mariadb/Dockerfile

```diff
@@ -22,5 +22,7 @@ RUN --mount=type=cache,id=mariadb-apk-${TARGETARCH},sharing=locked,target=/var/c
 # base image. Set to 10 minutes just incase it ran on very old or overallocated
 # hardware.
 ENV S6_CMD_WAIT_FOR_SERVICES_MAXTIME=600000
+# Default value of 256 MB in bytes
+ENV DB_MAX_ALLOWED_PACKET=268435456
```

Comment:
Please change to the following, and remove the comment.

```bash
ENV MYSQL_MAX_ALLOWED_PACKET=16777216
```

Prefix to match the other environment variables provided by the image, see the [README.md](https://github.com/Islandora-Devops/isle-buildkit/tree/main/mariadb#database-settings).

Also, change the default value to be the same default as is provided by MariaDB (`16777216`) to not change the existing behavior. See the documentation [here](https://mariadb.com/docs/server/ref/mdb/system-variables/max_allowed_packet/).

Reply:
Also add documentation for the new environment variable to the image's [README.md](https://github.com/Islandora-Devops/isle-buildkit/tree/main/mariadb#database-settings). 

For example:

| Environment Variable     | Default  | Description                                                                                                                                            |
| :----------------------- | :------- | :----------------------------------------------------------------------------------------------------------------------------------------------------- |
| MYSQL_ROOT_PASSWORD      |          | The database root user password. Defaults to `DB_ROOT_PASSWORD`                                                                                        |
| MYSQL_ROOT_USER          |          | The database root user (used to create the site database). Defaults to `DB_ROOT_USER`                                                                  |
| MYSQL_MAX_ALLOWED_PACKET | 16777216 | Max packet length to send to or receive from the server, [documentation](https://mariadb.com/docs/server/ref/mdb/system-variables/max_allowed_packet/) |